xml/System.Net.NetworkInformation/PhysicalAddress.xml

```diff
@@ -329,16 +329,20 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The `address` parameter must contain a string that can only consist of numbers and upper-case letters as hexadecimal digits. Some examples of string formats that are acceptable are as follows:  
+ The `address` parameter must contain a string that can only consist of numbers and letters as hexadecimal digits. Some examples of string formats that are acceptable are as follows:  
```

Comment:
```suggestion
 The `address` parameter must contain a string that can only consist of numbers and letters as hexadecimal digits. In .NET Framework and .NET Core 3.1 and earlier, the letters must be uppercase. Some examples of string formats that are acceptable are as follows:  
```

Reply:
The suggested change here doesn't seem necessary because these remarks are for the Parse(ReadOnlySpan) overload, which is only available in .NET 5 and later versions. I did however, make these suggested changes to the Parse(string) method so that the clarification was available in the remarks for that method. 

Let me know if this is incorrect, and the Parse(ReadOnlySpan) method still should have these clarifications.

Reply:
I see, you're correct!